nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -51,6 +52,41 @@ collision_monitor:
       min_points: 6
       visualize: False
       enabled: True
+    VelocityPolygonApproach:
+      type: "velocity_polygon"
+      action_type: "approach"
```

Comment:
Is this really using the approach model? I thought these were stop polygons? Is the approach type even possible with this velocity polygon? I'm pretty sure its just for stop / slowdown,